README.md

```diff
@@ -12,6 +12,11 @@ To monitor your source directory for changes, and recompile files if they change
 python build.py & python -m SimpleHTTPServer && fg
 ```
 
+For Python 3.6.0 : 
+```
+python build.py & python -m http.server && fg
```

Comment:
Sorry for the late response, but the above code should work for any versions > 3.3 since there have been no breaking changes after that.

So you can go ahead and change that